lib/authentication/screens/verify_email_screen.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class VerifyEmail extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return const Scaffold(
+      body: Center(
+        child: Text('Please verify email address')
+      )
+    );
+  }
+}
```

Comment:
Since this is not done. Should we create a new issue for resending email and such?

Reply:
Resend verification is part of the verification screen in my opinion, so that’s not necessary.